src/User32/User32+DeviceStateFlags.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+
+    /// <content>
+    /// Contains the <see cref="DeviceStateFlags"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// Device state flags.
+        /// </summary>
+        [Flags]
+        public enum DeviceStateFlags : uint
```

Comment:
Given the prefix in each constant, should we call this `DisplayDeviceFlags` instead?